src/Components/App/App.js

```diff
@@ -61,30 +64,23 @@ function App() {
     <div className="App">
       <header>
         <h1 className="header__h1">Selector</h1>
-        <Search
-          setTitles={ setTitles }
-        />
+        <Search setSearchArtist={setSearchArtist}/>
       </header>
       <Navigation isMobile={ isTabletOrMobile }/>
       <main>
         <Switch>
           <Route exact path="/">
-            <AlbumCardsDisplay
-              titles={ titles }
-              favorites={ favorites }
-              toggleFav={ toggleFav }
-              determineFav={ determineFav }
-              isFavorite={ isFavorite }
-            />
+            <section className="glass">
```

Comment:
I removed this 'section' element and put the glass class on the AlbumCardsDisplay component to streamline as its being applied to all instances on an old branch that won't be merged, would reccomend doing it here as well

Reply:
Done